packages-common/piw-utils/package.json

```diff
@@ -6,10 +6,10 @@
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
     "scripts": {
-        "start": "tsc --watch",
+        "start": "..\"/../node_modules/.bin/tsc\" --watch",
         "prebuild": "rm -rf ./dist",
-        "build": "tsc",
-        "lint": "eslint --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
+        "build": "..\"/../node_modules/.bin/tsc\"",
+        "lint": "..\"/../node_modules/.bin/eslint\" --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
```

Comment:
`../../.eslintrc.js` doesnt need \" ?

Reply:
just the first argument executable

Reply:
Is just for the executable. If you put outside "" it will say command .. not found on windows